src/zcbenchmarks.cpp

```diff
@@ -31,6 +31,31 @@
 #include "zcash/IncrementalMerkleTree.hpp"
 
 using namespace libzcash;
+// This method is based on Shutdown from init.cpp
+void shutdown_wallet()
+{
+    LogPrintf("%s: In progress...\n", __func__);
+    static CCriticalSection cs_Shutdown;
+    TRY_LOCK(cs_Shutdown, lockShutdown);
+    if (!lockShutdown)
+        throw new std::runtime_error("Failed to lock in shutdown_wallet");
+
+    if (pwalletMain)
+        pwalletMain->Flush(false);
+    #ifdef ENABLE_MINING
+        GenerateBitcoins(false, NULL, 0);
+    #endif
```

Comment:
Ah, misunderstanding here. This should all be indented one-less, because the convention is for `#ifdef`s to have no indentation (unless there is obvious nearby nesting), while the `GenerateBitcoins()` at this point makes it look like it is part of the if block, which it isn't.

Reply:
Changed.